docs/topics/installing-web-console-on-openshift.adoc

```diff
@@ -40,8 +40,8 @@ endif::[]
 ifdef::ocp-42[]
 .Procedure
 
-. Navigate to the link:{MTADownloadPageURL}[{ProductShortName} Download page] and download the {WebName} `Local install & OpenShift` file.
-. Extract the `.zip` file to a directory, for example, `MTA_HOME`.
+. Navigate to the link:{{ProductShortName}DownloadPageURL}[{ProductShortName} Download page] and download the {WebName} `Local install & OpenShift` file.
```

Comment:
You are checking in master.adoc (after pressing Ctrl+Shift+A) that there is a link here, right? Please be sure you have seen that all documents render correctly by using this tip -- or by checking the deploy preview. That's why you want to have one PR per document or per chapter even -- so you and others can read the rendered text, not just the files.